interactions/api/models/channel.py

```diff
@@ -183,13 +183,20 @@ def __init__(self, **kwargs):
 
     async def send(
         self,
-        content: Optional[str] = None,
+        content: Optional[str] = MISSING,
         *,
-        tts: Optional[bool] = False,
+        tts: Optional[bool] = MISSING,
         # attachments: Optional[List[Any]] = None,  # TODO: post-v4: Replace with own file type.
-        embeds=None,
-        allowed_mentions=None,
-        components=None,
+        embeds: Optional[Union["Embed", List["Embed"]]] = MISSING,  # noqa
+        allowed_mentions: Optional["MessageInteraction"] = MISSING,  # noqa
+        components: Optional[
+            Union[
+                "ActionRow",  # noqa
+                "Button",  # noqa
+                "SelectMenu",  # noqa
+                List[Union["ActionRow", "Button", "SelectMenu"]],  # noqa
```

Comment:
Not requesting, but, is `List[Union[...]]` proper? My understanding of `List` was that you could only apply one type in it. 

![image](https://user-images.githubusercontent.com/41456914/151628707-d444ccbe-bc67-4b76-ad07-767b11bc7ead.png)

Reply:
I don't know, I'm not familiar enough with it, in my opinion `List[Union[int, str]] ` is the same as `List[int], List[str]`

Reply:
I like the idea of explicitly mentioning what can be a list by singular types. Doing `List[Union[...]]` can seem a little confusing to some because it's a repeat of the base `Union`. This also doesn't seem to follow how `GenericAlias` works in the typing module upon closer investigation, so if we were to ever implement reading from the argument signatures' types in the future, it wouldn't work correctly.

My recommendation is switching over to `List[X], List[Y], ...` in the base `Union` structure.

Reply:
Could you request it as a change then? It's easier to just accept it from mobile than to change it

Reply:
Hm, I'm gonna change it, let's see if I'm good enough for pre-commit

Reply:
OK, I'm done now with that issue

And I managed to make the checks pass